receiver/kafkametricsreceiver/README.md

```diff
@@ -35,6 +35,7 @@ Metrics collected by the associated scraper are listed [here](metadata.yaml)
 
 Optional Settings (with defaults):
 
+- `cluster_alias`: Alias name of the cluster. Adds `cluster_alias` attribute to all the metrics.
```

Comment:
it's added as _resource_ attribute

Reply:
revised the language let me know if this doesn't work.